### DISCUSSION TITLE
Snaking in/out sliders should be flipped to match stable

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#21493] [Q&A]
In stable, the snaking sliders setting says that sliders 'snake out' from their starting point:

![image](https://user-images.githubusercontent.com/74088315/205219344-47012c3a-727b-47e8-a3a3-a1172877a75e.png)

However, in lazer, the setting that does the equivalent of this and causes sliders to 'snake out' from their starting point is actually called 'snaking in sliders', which is very confusing. The 'snaking out sliders' setting in lazer is actually the new setting which causes sliders to get smaller as they are played and 'snake in' towards the end of the slider. To put it simply, I think that it would make more sense if the names of these settings were flipped.

**Comment by peppy** [comment]
Maybe we can change the terminology to remove any chance of misunderstanding

"Snake sliders during appearing"
"Snake sliders during disappearing"

.. I'm hoping someone can come up with something better.

**Reply by smoogipoo** [reply]
"snake sliders to appear/disappear", not sure.

**Reply by monochrome22** [reply]
Outward snaking sliders
Inward snaking sliders

Snake appearing sliders
Snake disappearing sliders

It's better to keep the word "snaking", because that's what stable users would search. If it's changed to "snake", then the options should also appear when searching "snaking".
